### DISCUSSION TITLE
points capture while moving around

### DISCUSSION THREAD

**Discussion by frazucca** [opengisch/QField#2290] [Q&A]
Hi at all,
I wonder if it's possible in QField to trigger automatically points capture, with  both a spatial &/or temporal threshold, while the operator is moving around on the field.
Any hints are welcomed

TIA
ciao
franz

**Comment by marioba** [comment]
Hello @frazucca,
at the moment we do have tracking in QField but only with lines layers. We are currently looking for sponsors to implement it with point layers too.
